workflows/iuf/operations/management-nodes-rollout/management-m001-rollout.yaml

```diff
@@ -170,14 +170,14 @@ spec:
 
                     source /etc/cray/upgrade/csm/myenv
                     scp ncn-m001:/root/csm_upgrade.pre_m001_reboot_artifacts.*.tgz /root
-                    # Add a check for the scp command result
-                    zypper --plus-repo="/etc/cray/upgrade/csm/csm-${CSM_RELEASE}/tarball/csm-${CSM_RELEASE}/rpm/cray/csm/sle-$(awk -F= '/VERSION=/{gsub(/["-]/, "") ; print tolower($NF)}' /etc/os-release)" --no-gpg-checks install -y cray-site-init
+                    zypper --plus-repo="${CSM_ARTI_DIR}/rpm/cray/csm/sle-$(awk -F= '/VERSION=/{gsub(/["-]/, "") ; print tolower($NF)}' /etc/os-release)" --no-gpg-checks install -y cray-site-init
                     scp ncn-m001:/root/*.noarch.rpm /root/
-                    # Add a check for the scp command result
-                    rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
-
-                    rpm -Uvh --force /root/libcsm-latest.noarch.rpm
-
+                    if [ -f /root/docs-csm-latest.noarch.rpm ]; then
```

Comment:
Does an else statement need to be added if this file does not exist? In what case do we not expect to have the 'docs-csm-latest.noarch.rpm' file?

Reply:
Added the else statement with the required ERROR echo statements